src/vs/workbench/contrib/chat/browser/chatAccessibilityProvider.ts

```diff
@@ -24,7 +25,7 @@ export class ChatAccessibilityProvider implements IListAccessibilityProvider<Cha
 	}
 
 	getRole(element: ChatTreeItem): AriaRole | undefined {
-		return 'listitem';
+		return isMacintosh ? 'listitem' : 'document';
```

Comment:
> For mac, using document vs listitem would limit navigation flexibility

What do you mean by that? voiceover handles it differently?

I'll yield to your judgement for this, but I thought that we generally don't use 'document' because of the downsides you mention. Do we use it in other places in VS Code?

Could you add a comment about this in the code?

Reply:
Using document role is too restrictive with voiceover. It would make navigation within the element challenging.

Reply:
I plan to talk through this with JooYoung tomorrow